dbus.go

```diff
@@ -288,3 +332,74 @@ func (ss *SecretService) attrsMatch(attrs map[string]string, itemPath dbus.Objec
 	}
 	return true, nil
 }
+
+func (ss *SecretService) unlockItem(itemPath dbus.ObjectPath) (dbus.ObjectPath, error) {
```

Comment:
Rework as `unlockObject` and use in `ss.Store(...)`

Reply:
Done.